src/sentry/api/bases/sentryapps.py

```diff
@@ -278,8 +277,7 @@ def has_object_permission(self, request: Request, view, organization):
 
         self.determine_access(request, organization)
```

Comment:
Learning question: What side effect does this call have? We don't return anything, so by the interface, it implies it's updating either the request/org/self object somehow. 

Any way we can make it not have a side effect, and return something instead?

Reply:
The access file is a whole thing we have for our request authentication.
We do a bunch of checks, raise if necessary, and  set `request.access` on the passed in request.
I think we _could_ have it return the access instead, but we'd have to check with hybrid cloud to make sure that's ok